pkg/ebpf/probes/lsm.go

```diff
@@ -3,7 +3,7 @@ package probes
 import (
 	bpf "github.com/aquasecurity/libbpfgo"
 
-	"github.com/aquasecurity/tracee/pkg/errfmt"
+	"github.com/aquasecurity/tracee/common/errfmt"
```

Comment:
This should be amended to the commit that introduced it so it will compile correctly